tests/EmptyBasicFieldsTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use Corcel\Acf\Field\Text;
+use Corcel\Acf\Field\BasicField;
+use Corcel\Post;
+
+/**
+ * Class BasicFieldTest.
+ *
+ * @author Junior Grossi <juniorgro@gmail.com>
+ */
+class EmptyBasicFieldsTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Post
+     */
+    protected $post;
+
+    /**
+     * Setup a base $this->post object to represent the page with the basic fields.
+     */
+    protected function setUp()
+    {
+        $this->post = Post::find(82); // it' a page with empty custom fields
+    }
+
+    public function testTextFieldValue()
+    {
+        $text = new Text();
+        $text->process('fake_text', $this->post);
```

Comment:
Yep! I had to change the class signature to make possible to pass the `connection` to others instances.

Can you change those tests to this?

```
$text = new Text($this->post);
$text->process('fake_text');
```